Objective-C/Internal/Replicator/CBLStoppable.h

```diff
@@ -0,0 +1,31 @@
+//
+//  CBLStoppable.h
+//  CouchbaseLite
+//
+//  Copyright (c) 2020 Couchbase, Inc All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol CBLStoppable <NSObject>
+
+- (void) _stop;
```

Comment:
Is there a reason that this cannot be `stop`?

Reply:
public interface declaration will be an extra declaration(only for user reference). Not sure, whether that cause any warnings/compile issue.

Reply:
I think it shouldn't cause the compilation problem. If there is no logic problem to use stop, I think we can just use stop.

Reply:
that will make code more cleaner, the overhead of calling `_stop` from `stop`. let me try that.